opsdroid/connector/slack/__init__.py

```diff
@@ -74,12 +76,16 @@ def __init__(self, config, opsdroid=None):
 
     async def disconnect(self):
         """Disconnect from Slack."""
-        await self.slacker.close()
+        with contextlib.suppress(TypeError):
```

Comment:
We are already catching some errors higher up and trying to reconnect. Perhaps we should just catch this there too.